### PR TITLE
Suggestion to adding conversion functions from and to 64 bit float value

### PR DESCRIPTION
**Pull request by Silfurion**
Hello,

With the rise of ARMv8 processors, the use of the float64 bit version of CMSIS-DSP functions will become more and more frequent. Following this logic it will also be more common to make conversions to or from float64 type. Moreover in our projects we have to be able to convert from int to float64 and vice versa.

It is in this context that we would like to suggest new functions for conversion to or from float64 bit type inspired by float32 bit versions. 

We thank you in advance and wish you a good day

### CODE REVIEW

File: Source/SupportFunctions/arm_f64_to_q15.c [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  blkCnt = blockSize >> 2U;
+
+  while (blkCnt > 0U)
+  {
+    /* C = A * 32768 */
+
+    /* convert from float to Q15 and store result in destination buffer */
+#ifdef ARM_MATH_ROUNDING
+
+    in = (*pIn++ * 32768.0f);
```
Comment (christophe0606) [inline Source/SupportFunctions/arm_f64_to_q15.c:86]: Don't use float constant in a float64 code.

File: Source/SupportFunctions/arm_f64_to_q7.c [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+ * Title:        arm_f64_to_q7.c
+ * Description:  Converts the elements of the 64 bit floating-point vector to Q7 vector
+ *
+ * $Date:        18 August 2022
+ * $Revision:    V1.0.0
+ *
+ * Target Processor: Cortex-M and Cortex-A cores
+ * -------------------------------------------------------------------- */
+/*
+ * Copyright (C) 2010-2021 ARM Limited or its affiliates. All rights reserved.
```
Comment (christophe0606) [inline Source/SupportFunctions/arm_f64_to_q7.c:12]: Should be 2022. 
I have not looked at other files to check if there is a similar issue.

**Review by christophe0606 — CHANGES_REQUESTED** [review]
Minor issues with float constants which should be double constants.
I'll need to write tests for those new functions. Writing tests takes time ... so this PR won't be merged quickly. I'll do my best.

**Comment by jbr-smtg** [comment]
Bonjour @christophe0606, most issues should be fixed now. Let me know if you need further fixes.

**Comment by christophe0606** [comment]
I've finally added the tests, cleaned and  merged.

**Pull request closed without merging**